docs/user-guide/third-party-addons/opentelemetry.md

```diff
@@ -165,17 +162,11 @@ To install the example and collectors run:
       kind: ConfigMap
       data:
         enable-opentelemetry: "true"

```

Comment:
I'd rather keep the sample configmap as it is. It would make sense those who visit this page, also know about the options before going through the configs on the other pages.